src/patterns/dosdp-patterns/blood_vessel_pattern.yaml

```diff
@@ -0,0 +1,78 @@
+pattern_name: blood_vessel_pattern
+pattern_iri: http://purl.obolibrary.org/obo/uberon/patterns/blood_vessel_pattern.yaml
+description: "This is a pattern for a blood vessel."
+
+
+classes:
+  blood vessel: "UBERON:0001981"
+
+relations:
+  connecting branch of: "RO:0002252"
+  tributary of: "RO:0002376"
+  vessel supplies blood to: "RO:0020101"
+  vessel drains blood from: "RO:0020102"
+  part of: "RO:0000050"
+
+annotationProperties:
+  obo_foundry_uniquename: "http://purl.obolibrary.org/obo/IAO_0000589"
+  hasExactSynonym: "oboInOwl:hasExactSynonym"
+  hasDbXref: "oboInOwl:hasDbXref"
+
+vars:
+  parent: ""
```

Comment:
Why  not typing these? Why "" instead of, say, "anatomical entity"?

`parent` is not a name for a slot in a dosdp pattern. Sounds again like a ROBOT template to me.

Reply:
You can call your slots whatever you want, but it is good practice to specify a range.

Reply:
I was adapting the [design pattern that we have for the VCCF ontology](https://github.com/obophenotype/vccf-ontology/blob/main/src/patterns/dosdp-patterns/vessel.yaml), but yes, I should have added a range.